drake/systems/Simulation.h

```diff
@@ -103,8 +103,9 @@ inline bool handle_realtime_factor(const TimePoint& wall_clock_start_time,
 template <typename System>
 void simulate(const System& sys, double ti, double tf,
               const typename System::template StateVector<double>& xi,
-              SimulationOptions& options) {
-  if (options.realtime_factor < 0.0) options.realtime_factor = 0.0;
+              const SimulationOptions& options) {
+  SimulationOptions my_options = options;
```

Comment:
Consider: rename the arg to be input_options and keep the local as "options"... possible hazard that anyone editing this method would accidentally use "options.something" and not realize that my_options was the preferred spelling within the method.